### ISSUE TITLE
Checkbox no longer supports HTML labels in Play 2.7

### ISSUE THREAD

**Issue by Shnick** [playframework/playframework#9361]
This use to work in Play 2.6.x but does not work in Play 2.7.x. Can't spot any documentation saying this is no longer supported so assuming it's a bug?

Page looks like
```scala 
@import helper._
@checkbox(myForm("checkbox"),
  '_label -> Html(Messages("checkbox.label")),
  '_help  -> "")
```
Messages looks like
```properties
checkbox.label=I agree to the following <a href="terms.html" target="_blank">Terms and Conditions</a> .
```

Expect to see a check box with a click able link.  E.G
> [] I agree to the following <a href="terms.html" target="_blank">Terms and Conditions</a>

Instead I'm seeing HTML on the page. E.G
> [] I agree to the following \<a href="terms.html" target="_blank">Terms and Conditions</a>

**Comment by mkurz** [comment]
Seems to be related to #8656. I will have a look.

**Comment by mkurz** [comment]
I know what the problem is and will provide a pull request asap.

**Comment by Shnick** [comment]
Thanks for looking into this so quickly :+1:

**Comment by mkurz** [comment]
Fix is here #9385
